### ISSUE TITLE
Continuing text after a Smart Character pair

### ISSUE THREAD

**Issue by sqril** [writemonkey/wm3#345]
I really like the smart characters but I don't know how to "enter" them.  I have to go to my mouse, click outside the character, and then go back to typing which breaks the flow somewhat.

I've tried Tab and Enter and they just tab and enter - am I missing something? If so, what is it!!!

**Comment by eyeamonlyme** [comment]
I think this problem just comes with the territory. The major benefit of the smart character tool is that one can go back into previously composed text or code and easily bracket a selection. 

However, this means one must get used to using the arrow key to move the cursor outside the automatically generated second "smart" character. One may see this as a break in flow, but consider that one would have to break flow slightly in order to insert special characters such as ", [], (), or * anyway. 

In other words just make an effort to learn the motion of striking the arrow key, developing a habit, making it second nature and all will be well.
